data/dashboard.yml

```diff
@@ -41,6 +41,17 @@
         - name: Repositories
           url: /repos.html
           description: All the applications and other repositories we use to build GOV.UK.
+    - name: GOV.UK documentation
+      sites:
+        - name: 2nd-line Support Manuals
+          url: /manual/2nd-line.html
+          description: The documentation for GOV.UK's "Second-line Support"
```

Comment:
Just wondering how we know this a common link would be useful here compared to any other part of dev docs?